src/components/tree-views-container.tsx

```diff
@@ -115,7 +120,7 @@ function updatedTree(prevRoots, prevMap, nodeId, children) {
             }
         });
 
-    const prevChildren = nodeId ? prevMap[nodeId]?.children : prevRoots;
+    const prevChildren = nodeId && prevMap?.[nodeId].children ? prevMap[nodeId].children : prevRoots;
```

Comment:
This changes the behavior why don't you keep previous code ?

Reply:
Done